ios-base/DroidKaigi 2020/Session/Views/SpeakerViewController.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import ios_combined
+import Nuke
+import UIKit
+
+final class SpeakerViewController: UIViewController {
```

Comment:
Please move this file to new directory. Such as `$SRCROOT/Droid Kaigi/Speaker/Views`.

Reply:
oh sorry, i misunderstood project structures